src/components/Item/ItemCard.jsx

```diff
@@ -0,0 +1,43 @@
+import heart_icon from "/heart_icon.svg";
+
+import "./ItemCard.css";
+
+function isValidImage(url) {
+  return (
+    typeof url === "string" &&
+    url.startsWith("http") &&
+    !url.includes("via.placeholder.com")
+  );
+}
+
+function ItemCard({ item, className }) {
+  return (
+    <>
+      <div className="item-card-container">
+        <img
+          src={isValidImage(item.images[0]) ? item.images[0] : "/no_image.png"}
+          onError={(e) => {
+            e.target.onerror = null;
+            e.target.src = "/no_image.png";
+          }}
+          alt={item.name}
+          className={`${className} item-image`}
+        />
+        <div className="item-card-description">
+          <span className="item-name">{item.name}</span>
+          <h2 className="item-price">{item.price}</h2>
```

Comment:
💊 제안
금액의 경우 (,)를 통해 단위가 구분되도록 아래처럼 작성해주세요!

```suggestion
          <h2 className="item-price">{item.price.toLocaleString()}</h2>
```